### PR TITLE
More than cat facts by now

### PR DESCRIPTION
**Pull request by ACA30**
Can't have enough.

### CODE REVIEW

**Comment by ACA30** [comment]
Just added a bunch more crap :3

**Comment by ACA30** [comment]
At this point, you should pretty much merge this ASAP, as several crucial typos were fixed

File: docs/templates/ads.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-I rent out the lower left corner to help pay for the cost of running yagpdb, if you want to rent that spot then feel free to contact me, either on email (jonasr747@gmail.com) or discord.
+I rent out the lower left corner to help pay for the cost of running yagpdb. If you want to rent that spot, then feel free to contact me, either on email (jonasr747@gmail.com) or on Discord (jonas747#0001).
```
Comment (jonas747) [inline docs/templates/ads.md]: Capital J, as discord usernames are case sensitive

File: docs/templates/helping-out.md [hunk 2]
```diff
@@ -47,8 +47,8 @@ Check the guide below if you're interested and if you have any questions then as
 
 ### Contributing code
 
-The bot is written in go, and the web interface is using plain HTML with bootstrap and some small javascript here and there. If you want to make a contribution like fix a bug, add a feature and so on you should probably notify me first by making a issue or sending me a message on discord so that you don't waste your time in case I was also working on that.
+The bot is written in go, and the web interface is using plain HTML with Nootstrap and some small JavaScript here and there. If you want to make a contribution like fix a bug, add a feature and so on you should probably notify me first by making an issue, or sending me a message on Discord so that you don't waste your time in case I was also working on that.
```
Comment (jonas747) [inline docs/templates/helping-out.md]: Nootstrap?

File: docs/templates/index.md [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
-It is currently very incomplete and very work in progress, so if you want to help out writing docs please don't hesitate to do so.
+It is currently very incomplete and very work in progress, so if you want to help out writing docs, please don't hesitate to do so.
 
-If you don't find the information you're looking for then i recommend you to join the support server above.
+If you don't find the information you're looking for, then I recommend you join the support server, which can be found above.
 
 ## Donating
 
-If you want to donate or [buy the ad-space](/docs/ads) you can do so using patreon, bitcoin, bitcoincash, etherum and litecoin:
+If you want to donate or [buy the ad-space](/docs/ads) you can do so using Patreon, Bitcoin, Bitcoincash, Etherum and Litecoin:
```
Comment (jonas747) [inline docs/templates/index.md]: should be Ethereum

File: stdcommands/catfacts.go [hunk 4]
```diff
@@ -213,4 +213,33 @@ var Catfacts = []string{
...
+	`Female cats are typically right-pawed while male cats are typically left-pawed.`,
+	`Basically, cats have a lower social IQ than dogs but can solve more difficult cognitive problems when they feel like it.`,
+	`Cats are magical. Here's a cute cat for you. :3 - https://giphy.com/gifs/JIX9t2j0ZTN9S`,
+	`Cats have 1,000 times more data storage than an iPad.`,
+	`Isaac Newton is credited with inventing the cat door.`,
+	`A house cat is faster than Usain Bolt.`,
+	`When cats leave their poop uncovered, it is a sign of aggression to let you know they don't fear you.`,
+	`Cats use their whiskers to detect if they can fit through a space.`,
+	`A cat's nose is ridged with a unique pattern, just like a human fingerprint.`,
+	`Cats lick themselves to get *your* scent off, not theirs... Nice try though...`
```
Comment (jonas747) [inline stdcommands/catfacts.go]: Add a trailing comma

File: stdcommands/general_commands.go [hunk 5]
```diff
@@ -533,7 +533,7 @@ func HandleMessageCreate(evt *eventsystem.EventData) {
 	taken := time.Duration(time.Now().UnixNano() - parsed)
 
 	started := time.Now()
-	common.BotSession.ChannelMessageEdit(m.ChannelID, m.ID, "Gatway (http send -> gateway receive time): "+taken.String())
+	common.BotSession.ChannelMessageEdit(m.ChannelID, m.ID, "Gateway (http send -> gateway receive time): "+taken.String())
 	httpPing := time.Since(started)
 
 	common.BotSession.ChannelMessageEdit(m.ChannelID, m.ID, "HTTP API (Edit Msg): "+httpPing.String()+"\nGatway: "+taken.String())
```
Comment (jonas747) [inline stdcommands/general_commands.go]: The same typo is down here aswell if you wanna fix that

File: docs/templates/helping-out.md (hunk 2 above)
Comment (ACA30) [reply docs/templates/helping-out.md]: LMAO I just saw that, haha...... I'm fixing everything now

File: docs/templates/ads.md (hunk 1 above)
Comment (ACA30) [reply docs/templates/ads.md]: Fixed

File: docs/templates/index.md (hunk 3 above)
Comment (ACA30) [reply docs/templates/index.md]: Fixed

File: stdcommands/catfacts.go (hunk 4 above)
Comment (ACA30) [reply stdcommands/catfacts.go]: k

File: stdcommands/general_commands.go (hunk 5 above)
Comment (ACA30) [reply stdcommands/general_commands.go]: Hmmm, I was sure I fixed that first lol... Everything is fixed now. :)

**Comment by ACA30** [comment]
All the changes should be completed.

File: moderation/assets/moderation.html [hunk 6]
```diff
@@ -1,4 +1,4 @@
-{{define "template_helper_mod_author"}}<code>{{"{{"}}.Author.(Username/ID/Discriminator){{"}}"}}</code> - The author of the kick/ban{{end}}
+{{define "template_helper_mod_author"}}<code>{{"{{"}}.Author.(Username/ID/Disciminator){{"}}"}}</code> - The author of the kick/ban{{end}}
```
Comment (jonas747) [inline moderation/assets/moderation.html]: Discriminator was right

Comment (ACA30) [reply moderation/assets/moderation.html]: Oh.... LOL

**Comment by ACA30** [comment]
Done.

**Pull request merged**